web/src/PickNeighbourhoodMode.svelte

```diff
@@ -170,19 +169,6 @@
         </li>
         <li>
           {pageTitle($mode.mode)}
-          <HelpButton>
```

Comment:
There were just two modes with these. I got rid of it for now. Maybe in the future, we could try putting the help button over here again, and making it open to the appropriate section of the manual